src/main/java/com/microsoftopentechnologies/windowsazurestorage/helper/AzureUtils.java

```diff
@@ -170,7 +174,7 @@ public static SharedAccessBlobPolicy generateBlobPolicy() {
      *
      * @param storageAccount
      * @param fileName
-     * @param shareName  container name
+     * @param shareName      container name
```

Comment:
You may want to update the doc to make it more professional by the way.

Reply:
Used IDE's `Reformat Code` function, better in separate PR next time.